support-lib/jni/djinni_support.cpp

```diff
@@ -28,6 +28,14 @@ namespace djinni {
 // Set only once from JNI_OnLoad before any other JNI calls, so no lock needed.
 static JavaVM * g_cachedJVM;
 
+template<>
```

Comment:
Can you add a comment explaining the rationale for this, similar to what's in this PR's description?